docs/reference/indices/create-index.asciidoc

```diff
@@ -104,24 +113,9 @@ curl -XPUT localhost:9200/test -d '{
             "routing" : "kimchy"
         }
     }
-}'
---------------------------------------------------
-
-[float]
-=== Creation Date
-
-When an index is created, a timestamp is stored in the index metadata for the creation date.  By 
```

Comment:
yes, the index creation timestamp cannot be set by users anymore, like the creation version